lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
-        <mat-datepicker-toggle matSuffix [for]="fromDatepicker" data-automation-id="date-range-from-date-toggle"></mat-datepicker-toggle>
-        <mat-datepicker #fromDatepicker color="accent"></mat-datepicker>
-        <mat-error *ngIf="from.invalid" data-automation-id="date-range-from-error">
-            {{ getFromValidationMessage() | translate: { requiredFormat: datePickerFormat } }}
-        </mat-error>
-    </mat-form-field>
-
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-TO-DATE' | translate }}</mat-label>
```

Comment:
please remove unused 'SEARCH.FILTER.RANGE.SELECT-TO-DATE' translation key from en.json file

Reply:
Removed unused translation keys